src/c4/C4_MPI_reductions_pt.cc

```diff
@@ -80,6 +80,8 @@ template void global_min<double>(double &);
 template void global_min<long double>(long double &);
 template void global_min<unsigned long long>(unsigned long long &);
 
+template void global_and<bool>(bool &);
```

Comment:
I like restricting this to `T=bool`. There might be an argument for using integral types, but let's try to avoid that.